docker/fedora.Dockerfile

```diff
@@ -1,9 +1,7 @@
 FROM fedora:latest
```

Comment:
I don't think we should provide such image, at least until someone asks for it :)

Reply:
Ah because it's used as a base for others

Reply:
The idea of ​​this image is that it be used as a base for the others, so that you do not have to download and install Flatpak and its dependencies for each image